docs/pages/x/api/data-grid/data-grid-premium.json

```diff
@@ -782,6 +782,12 @@
       "description": "Styles applied to the empty cell element.",
       "isGlobal": false
     },
+    {
+      "key": "cellOffsetLeft",
+      "className": "MuiDataGridPremium-cellOffsetLeft",
+      "description": "Styles applied to the left offset cell element.",
+      "isGlobal": false
+    },
```

Comment:
Tried to `@ignore - Do not document` this one but it didn't work. How can I do that?

Reply:
Indeed, it does not impact the classes.
I'll submit a PR to the core repo.

Reply:
https://github.com/mui/material-ui/pull/41009